src/Symfony/Component/HttpFoundation/HeaderBag.php

```diff
@@ -181,7 +182,7 @@ public function has($key)
      */
     public function contains($key, $value)
     {
-        return \in_array($value, $this->get($key, null, false));
+        return \in_array($value, $this->all((string) $key));
```

Comment:
Why the type cast here?

Reply:
to never pass `null`, and we don't want to get `string[][]` as result